.gitpod/readme.md

```diff
@@ -0,0 +1,49 @@
+# Welcome to Starport ✨
+
+Starport is an easy-to-use CLI tool for creating sovereign blockchains. Blockchains created with Starport use Cosmos SDK, the world's most widely used blockchain application framework.
+
+In this browser-based development environment, the terminal window is in the lower part of the window. The `starport` binary is pre-installed and ready to use on the command line.
+
+## Quick start
+
+To create a blockchain and start a node in development:
+
+```
+starport app github.com/user/hello
+
+cd hello
+
+starport serve
+```
+
+## Next steps
+
+📺 **[Introduction to Starport](https://www.youtube.com/watch?v=5RqAIE0b8Kw)**: watch an introductory video to learn about Starport.
```

Comment:
```suggestion
📺 **[Introduction to Starport](https://www.youtube.com/watch?v=5RqAIE0b8Kw)**: Watch an introductory video to learn about Starport.
```

Reply:
thanks for the explanation of this new and improved user experience in gitpod! Let's make this one last change. @fadeev